src/rules/no-important.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * @fileoverview Rule to disallow !important flags.
+ */
```

Comment:
Please add `@author` with your name and Yann Bertrand

Reply:
Thanks for the new PR 🙂 looks great

I'm happy if the first PR helped in any way (tests?), if not it's ok not to add me as coauthor.